PSKoans/ExportedTypes.ps1

```diff
@@ -11,4 +25,33 @@ class KoanAttribute : System.Attribute {
     KoanAttribute() {
         $this.Position = [uint32]::MaxValue
     }
+
+    [void] InvokeHelpInfo() {
+        if ($this.HelpAction) {
+            $this.InvokeHelpAction()
+        }
+        elseif ($this.HelpPath) {
+            $this.InvokeHelpPath()
+        }
+        elseif ($this.HelpURL) {
+            $this.InvokeHelpURL()
+        }
+    }
+
+    [void] InvokeHelpAction() {
+        $this.HelpAction.InvokeReturnAsIs()
+    }
+
+    [void] InvokeHelpURL() {
+        Write-ConsoleLine "Confusion is only the beginning"
+        Write-ConsoleLine "Your defeat has only made you stronger"
+        Write-ConsoleLine ("Learn through Zen: {0}" -f $this.HelpURL)
```

Comment:
Should we provide an `-Online` switch to mimic the Get-Help parameter and simply invoke that directly rather than giving them a relatively unfriendly URL?

Or do you think having the URL is better? Should we provide for the ability to specify multiple URLs, for example to provide both official help docs and comprehensive blog posts and writeups?